index.d.ts

```diff
@@ -0,0 +1,24 @@
+export interface Options<Default = null> {
+	/**
+	 * Use a key distance-based score to leniently accept typos of `yes` and `no`
+	 *
+	 * @default false
+	 */
+	lenient?: boolean;
+
+	/**
+	 * Default value if no match was found
+	 *
+	 * @default null
+	 */
+	default?: boolean | Default;
```

Comment:
No need for generics, it can only either be a boolean or null.

```suggestion
	default?: boolean | null;
```